liquid_node/commands.py

```diff
@@ -450,6 +474,110 @@ def deletecollection(name):
     purge_collection(name)
 
 
+def importcollection(name, path, method='copy'):
+    """Import a single collection extracted collection.
+    The path must contain:
+    1. The blobs folder, named `blobs`
+    2. The database folder, named `pg`, containing a subfolder called `data`.
+    3. the index, named <name>-index.tgz
+
+    :param name: name of the collection you want to import
+    :param path: path to the collection data.
+    :param method: can be 'copy', 'move' or  'link'. Copy is default.
+    """
+    path = Path(path).resolve()
+    database = path / 'pg'
+    blobs = path / 'blobs'
+    index = path / f'{name}-index.tgz'
+    log.info(f'Importing collection {name}')
+    # parsing `method`, which is not interpreted as a kwarg
+    if method != 'copy':
+        _, method = method.split('=', 1)
+        if method not in ['copy', 'move', 'link']:
+            raise RuntimeError(f'unknown method: {method}')
+
+    node_name = name.lower()
+    if name != node_name:
+        log.info(f'Renaming "{name}" to "{node_name}"')
+    collection_path = Path(config.liquid_volumes) / 'collections' / node_name
+    if collection_path.exists():
+        if confirm(f'collection {name} already exists, overwrite (hoover needs to be running)?'):
+            redeploy = True
+            purge_collection(name)
+        else:
+            log.info(f'exiting and not overwriting data for {name}')
+            exit()
+    else:
+        redeploy = False
+        (collection_path).mkdir(parents=True)
+
+    # copy the pg dir
+    database = Path(database).resolve(strict=True)
+    if not (database / 'data' / 'PG_VERSION').exists():
+        raise RuntimeError("database is not a valid Postgres database")
+    pg_src = database
+    pg_dst = collection_path / 'pg'
+    import_dir(pg_src, pg_dst, method=method)
+
+    # copy the blobs dir
+    blobs = Path(blobs).resolve(strict=True)
```

Comment:
`blobs` is already a `Path` object :) And do you really need `strict`? I had to look up the documentation to see what it means, imho it just adds mental overhead when reading the code. Also I suspect that calling `resolve` is ultimately not needed because all filesystem operations will work through symlinks anyway.

Reply:
Oh yeah it is, that was a change that came later on, thank you. 
So my ingenious idea of using strict was that it would bail out if the part doesn't exist and therefore slims the code, because I don't have to check whether the user has actually provided a legit path. Is that unnecessary or would you have done it differently?